docs/src/docs/asciidoc/user/writers.adoc

```diff
@@ -98,3 +96,29 @@ include::{sourcedir36}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[t
 <5> Define the concurrency level of write-behind queue(s).
 This indicates how many writer threads work in parallel to update the underlying system of record asynchronously.
 <6> Enable the write coalescing behavior, which ensures that only one update per key per batch reaches the underlying system of record.
+
+== Cache-Through for Clustered Cache
+
+A clustered cache can also be configured with a `CacheLoaderWriter` for a cache-through use case.
+
+[source%nowrap,java,indent=0]
+----
+include::{sourcedir36}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/ClusteredLoaderWriter.java[tag=clusteredWriteThroughCache]
+----
+
+<1> We create a sample `CacheLoaderWriter` with a known mapping for key `4L`.
+<2> Now add loader-writer to the `CacheConfiguration`.
+<3> We can do normal cache operations now.
+<4> We can observe that even if the cache had no mapping for key `4L`, it will still return the mapping for the key & populate the cache as well.
+
+=== Adding Write-Behind to Clustered Cache
+
+[source%nowrap,java,indent=0]
+----
+include::{sourcedir36}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/ClusteredLoaderWriter.java[tag=clusteredWriteBehindCache]
+----
+
+<1> Similar to non-clustered caches, here also we need a configured `CacheLoaderWriter`.
+<2> Additionally, register a `WriteBehindConfiguration` on the cache by using the `WriteBehindConfigurationBuilder`.
+
+NOTE: `BatchedWriteBehindConfigurationBuilder` configurations are not honoured for clustered caches yet.
```

Comment:
This can be rephrased as `Batching configuration is not honoured *by* clustered caches"